app/helpers/layout_helper.rb

```diff
@@ -8,36 +8,74 @@ def back_to_model_link
     end
   end
 
-  def country_option(code, opts = {})
+  def current_area_name
+    code = Current.setting.area_code
+    I18n.t(code, default: code.humanize)
+  end
+
+  def current_area_info
+    area_name = current_area_name
+    capture_haml do
+      unless Current.setting.derived_dataset?
+        haml_tag 'img',
+          src: icon_for_area_code(Current.setting.area_code),
+          title: area_name,
+          width: 16
+      end
+      unless Current.setting.area_code.length == 2
+        haml_concat area_name
+      end
+    end
+  end
+
+  def country_option(area)
+    code = area[:area_code]
+
     selector = params[:country] || Current.setting.area_code
     selected = selector == code ? "selected='true'" : nil
 
-    label = I18n.t("country_select.#{ code }", default: I18n.t(code))
-    label += " (#{ I18n.t('new') })" if opts[:test]
+    label = I18n.t("country_select.#{ code }", default: [code.to_sym, code.humanize])
+    label += " (#{ I18n.t('new') })" if area[:test]
 
-    content_tag :option, label.html_safe, :value => code, :selected => selected,
-      'data-earliest' => opts[:earliest] || 2013
+    content_tag :option, label.html_safe,
+      :value => code,
+      :selected => selected,
+      'data-earliest' => area[:analysis_year]
```

Comment:
I think [this should be `analysis_year + 1`](https://github.com/quintel/etmodel/pull/2313/files#diff-ccec77d3dd28c781f0ef06caa4e8c033L12). It doesn't make much sense to permit the `present` and the `future` to be the same year. I expect ETEngine will let you do it, but ETModel will produce confusing chart axes:

![screen shot 2016-12-13 at 12 51 10](https://cloud.githubusercontent.com/assets/4383/21141062/15226824-c133-11e6-8767-85d87e459d7e.png)